### DISCUSSION TITLE
Roadmap / Timeline

### DISCUSSION THREAD

**Discussion by jbcurler2010** [bitfocus/companion#2456] [General]
Is there a general roadmap/timeline for new version release?

**Comment by dnmeid** [comment]
No and yes. There is a general timeline where we want to release at least two stable updates every year. 3.0 was scheduled for late April / early May and now is a little delayed.
There is not a general roadmap at this point, but there are several individual projects developers are working on. The biggest try to create a roadmap is also here in the discussions (#2387), you are welcome to participate.

**Comment by jbcurler2010** [comment]
Thanks for the response. Very helpful.
